pjrt/pjrt.zig

```diff
@@ -140,13 +140,52 @@ pub const Api = struct {
         };
     }
 
+    pub inline fn stableHLOCurrentVersion(self: *const Api) ?[]u8 {
+        const state = struct {
+            var api: *const Api = undefined;
+            var buf: [5]u8 = undefined;
+            var once = std.once(fetch);
+
+            fn fetch() void {
+                if (api.getPluginAttribute("stablehlo_current_version")) |v| {
+                    const sliced_version = v.inner.unnamed_0.int64_array_value[0..v.inner.value_size];
+                    _ = std.fmt.bufPrint(&buf, "{d}.{d}.{d}", .{ sliced_version[0], sliced_version[1], sliced_version[2] }) catch unreachable;
```

Comment:
this will fail as soon as there is a two number version. Maybe bump it to a bigger buffer ?